src/manager.cpp

```diff
@@ -306,6 +306,9 @@ int Manager::updateKeyword(const types::Path i_vpdPath,
             return -1;
         }
 
+        // Get D-bus name for the given keyword
+        l_propertyName = vpdSpecificUtility::getDbusKeywordName(l_propertyName);
```

Comment:
You are passing a propertyName and receiving a property name, better name the API as "getDBusPropertyName"?
Anyways from DBus perspective there are properties not keywords.

Reply:
would agree

Reply:
done